web/content/release-3.4.adoc

```diff
@@ -0,0 +1,36 @@
+---
+title: 3.4
+---
+:FOREMAN_VER: 3.4
+:KATELLO_VER: 4.6
+
+== Foreman {FOREMAN_VER} and Katello {KATELLO_VER}
+
+Use the top menu bar for navigation for all guides.
+
+=== Quickstart
+
+* link:/{FOREMAN_VER}/Quickstart_Guide/index-foreman-el.html[Foreman on RHEL/CentOS Quickstart Guide]
+* link:/{FOREMAN_VER}/Quickstart_Guide/index-foreman-deb.html[Foreman on Debian/Ubuntu Quickstart Guide]
```

Comment:
Having these links implies it is supported, but the top bar navigation doesn't link it. I'm leaning to copying the 3.3 file rather than nightly for now.